scripts/bulk_update_leads_info.py

```diff
@@ -0,0 +1,105 @@
+#-*- coding: utf-8 -*-
+
+import re
+import argparse
+import csv
+import logging
+from closeio_api import Client as CloseIO_API, APIError
+
+get_contact_info = lambda key, row, what, typ: [{what: row[x], 'type': typ} for x in row.keys()
```

Comment:
seems like it would be more readable if it was a simple def function instead of lambda since it spans multiple lines